src/error.rs

```diff
@@ -50,14 +50,19 @@ pub enum Error {
     #[error("Transfer not expected for this actor {0:?}")]
     NoSetForTransferId(Dot<PublicKey>),
     /// Proposed operation is not the next in sequence. The debit op should be current actor count + 1
-    #[error("Operation out of order: debit's counter is '{0}', current actor counter is '{1}'")]
+    #[error(
+        "Operation out of order: debit's counter is '{0}', actor next expected op count is '{1}'"
+    )]
     OperationOutOfOrder(u64, u64),
     /// This account has not seen any debits yet. Sent debit should be 0 but was not.
     #[error("Operation out of order debit counter should be 0")]
     ShouldBeInitialOperation,
     /// No credits or debits were found to sync
-    #[error("No credits or debits found to sync to actor")]
-    NothingToSync,
+    #[error("There was no valid history found for provided actor.")]
+    NoActorHistory,
+    /// Credit history for the actor was invalid
+    #[error("Provided actor history could not be validated.")]
+    InvalidActorHistory,
```

Comment:
Is it meant only for Credit History? If not, we should update the doc.

Reply:
I'll update that aye